--- target/generated-sources/antlr3/cz/vutbr/web/csskit/antlr/CSSLexer.java	2013-09-16 15:25:14.000000000 +1000
+++ target/generated-sources/antlr3/cz/vutbr/web/csskit/antlr/CSSLexer.java	2013-09-16 15:25:39.000000000 +1000
@@ -5560,14 +5560,14 @@
 	static final char[] DFA45_max = DFA.unpackEncodedStringToUnsignedChars(DFA45_maxS);
 	static final short[] DFA45_accept = DFA.unpackEncodedString(DFA45_acceptS);
 	static final short[] DFA45_special = DFA.unpackEncodedString(DFA45_specialS);
-	static final short[][] DFA45_transition;
 
-	static {
+	static short[][] DFA45_transition() {
 		int numStates = DFA45_transitionS.length;
-		DFA45_transition = new short[numStates][];
+        short[][] DFA45_transition = new short[numStates][];
 		for (int i=0; i<numStates; i++) {
 			DFA45_transition[i] = DFA.unpackEncodedString(DFA45_transitionS[i]);
 		}
+        return DFA45_transition;
 	}
 
 	protected class DFA45 extends DFA {
@@ -5581,7 +5581,7 @@
 			this.max = DFA45_max;
 			this.accept = DFA45_accept;
 			this.special = DFA45_special;
-			this.transition = DFA45_transition;
+			this.transition = DFA45_transition();
 		}
 		@Override
 		public String getDescription() {
